.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: 'release'
+
+on:
+  push:
+    branches:
+    - 'main'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/checkout@v2'
+
+    - uses: 'actions/setup-node@v2'
+      with:
+        node-version: '16.x'
+
+    - name: 'npm build'
+      run: 'npm ci && npm run build && npm run docs'
+
+    - name: 'Create pull request'
+      uses: 'peter-evans/create-pull-request@dcd5fd746d53dd8de555c0f10bca6c35628be47a'
+      with:
+        token: '${{ secrets.ACTIONS_BOT_TOKEN }}'
+        add-paths: 'dist/'
+        committer: 'google-github-actions-bot <github-actions-bot@google.com>'
+        author: 'google-github-actions-bot <github-actions-bot@google.com>'
+        signoff: 'google-github-actions-bot <github-actions-bot@google.com>'
+        commit-message: 'Build dist'
+        title: 'chore: build dist'
+        body: 'Build compiled Typescript'
+        base: 'main'
+        branch: 'actions/build'
+        push-to-fork: 'google-github-actions-bot/setup-cloud-sdk'
+        delete-branch: true
+
+  # create-pull-request creates a release pull request if there are any
+  # convential commit changes since the last release.
+  create-pull-request:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'google-github-actions/release-please-action@v2'
+      with:
+        token: '${{ secrets.ACTIONS_BOT_TOKEN }}'
+        release-type: 'node'
+        bump-minor-pre-major: true
+        command: 'release-pr'
+        fork: true
+
+  # release does a release on the merge of the release pull request. It also
+  # updates the floating tag alias for the major version.
```

Comment:
I dont think we do this here

Reply:
The `updates the floating tag alias for the major version` part :), LGTM otherwise

Reply:
Removed.